README.md

```diff
@@ -379,31 +379,36 @@ ARGUMENTS
   COLLECTION  Collection name
 
 OPTIONS
-  --api-key=api-key        Kuzzle user api-key
-  --as=as                  Impersonate a user
-  --batch-size=batch-size  [default: 2000] Maximum batch size (see limits.documentsFetchCount config)
-  --editor                 Open an editor (EDITOR env variable) to edit the query before sending
+  --api-key=api-key          Kuzzle user api-key
+  --as=as                    Impersonate a user
+  --batch-size=batch-size    [default: 2000] Maximum batch size (see limits.documentsFetchCount config)
+  --editor                   Open an editor (EDITOR env variable) to edit the query before sending
 
-  --format=format          [default: JSONL] "jsonl or kuzzle - kuzzle will export in Kuzzle format usable for internal
-                           fixtures and jsonl allows to import that data back with kourou
+  --fields=fields            [CSV format only] The fields to be included in the CSV export in dot-path format (e.g.
```

Comment:
Those fields should be separated with a coma? If yes, you should update the example to be more clear

Reply:
ah yes you are right